### PR TITLE
Deeper Look into Git Lesson: Add clarity to rebase tool commands

### PR DESCRIPTION
**Pull request by EthanLeong97**
## Because

As a student myself, I was confused on how to navigate the rebase interactive tool. After some trial and error and Google I understood what may have been obvious. Hoping future students don't get tripped up on this as I did.

Also, I figured I would submit a PR in spirit of the next lesson!

## This PR

- specify how to save and exit the interactive tool
- specify how to name the squashed commit in the interactive tool

## Issue

Closes #XXXXX

## Additional Information

No hard feelings on closing the PR! If the intent was for me to figure out the tool, I'm better for it. Thank you!

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: git/intermediate_git/a_deeper_look_at_git.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ squash 92aa6f3 Create second file
 pick 05e5413 Create third file and create fourth file
 ~~~
 
-Rename the commit to `Create first and second file`, then finish the rebase. That's it! Run `git log` and see how the first two commits got squashed together.
+Rename the 1st commit message to `Create first and second file`, delete commit message #2, then finish the rebase. That's it! Run `git log` and see how the first two commits got squashed together.
```
Comment (linkonsat) [inline git/intermediate_git/a_deeper_look_at_git.md:99]: First change is definitely solid as it can be a bit confusing exiting the editor the first time. That said I don't think it's necessary to change the renaming the commit message sentence. That said, since when we are squashing we are combining the commits rather than deleting or replacing one. I do feel it would be better to keep it as is or just have the "Rename the commit" be "Change the commit message to". As that would capture the steps you're mentioning in essence rather than just applying the steps narrowly.

**Review by linkonsat — CHANGES_REQUESTED** [review]

File: git/intermediate_git/a_deeper_look_at_git.md (hunk 1 above)
Comment (EthanLeong97) [reply git/intermediate_git/a_deeper_look_at_git.md:99]: I see what you're saying about how I'm applying the steps narrowly and that implies we're deleting something. 

Changing the message of the newly squashed commit is definitely what I'm trying to convey here as it's not obvious how one would go about doing that. Given the instruction, one would assume that it becomes a single commit and we rename it in the same fashion as earlier in the lesson; however, the below editor is prompted after we exit the rebase tool changing pick to squash:
image(1)
Seeing the line "Please enter the commit message for your changes" led me to a bit of confusion on how I would go about that correctly.

Comment (linkonsat) [reply git/intermediate_git/a_deeper_look_at_git.md:99]: @EthanLeong97 Gotcha. The editor does explain how to make the changes a bit. If you have a different idea on how to rephrase it to make it clear were just replacing the commit message im more than happy to hear it. Maybe something like change "Rename the commit to'" with "Replace the old commit messages with". As it does become one commit and were just replacing the commit message. Kind of what we want to get at(In this section) is to explain the concept of what the person is doing rather than just the specific situational steps. Whereas in the other part the PR touched Git has very specific steps to exit the editor.

**Comment by KevinMulhern** [comment]
Hey @EthanLeong97, just checking in to see if you'd still like to work on this PR?

**Comment by KevinMulhern** [comment]
No response
